sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationAsyncClient.java

```diff
@@ -644,19 +484,19 @@ public Mono<GetLanguagesResult> getLanguages() {
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return the response body on successful completion of {@link Mono}.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<List<TranslatedTextItem>> translate(List<String> to, List<InputTextItem> requestBody,
-        String clientTraceId, String from, TextType textType, String category, ProfanityAction profanityAction,
-        ProfanityMarker profanityMarker, Boolean includeAlignment, Boolean includeSentenceLength, String suggestedFrom,
-        String fromScript, String toScript, Boolean allowFallback) {
+    private Mono<List<TranslatedTextItem>> translate(List<String> targetLanguages, List<InputTextItem> body,
```

Comment:
My understanding is that it would require TypeSpec change as some parameters named `body` are in generated methods. I can change not generated, but then it will not be consistent.